global/facebook-pixel@2.0.js

```diff
@@ -0,0 +1,12 @@
+if(typeof fbq === 'undefined' && document.location.href.search('gtm-msr.appspot') == -1) {
```

Comment:
```suggestion
if(typeof fbq === 'undefined' && document.location.origin.includes('gtm-msr.appspot')) {
```

Use `origin` to only include the first part, not the path. Use `includes` so you don't need to check against `-1`

Reply:
Thanks Jethro!
Didn't commit the suggestion because it should be the negative case:
 `if(typeof fbq === 'undefined' && !document.location.origin.includes('gtm-msr.appspot')) {`
 Resolving this in the next commit.

Reply:
Thanks, good check